### PR TITLE
Add fart

### PR DESCRIPTION
**Pull request by tmagicturtle**

### CODE REVIEW

File: data/mods/ssb/abilities.ts [hunk 1]
```diff
@@ -150,6 +150,60 @@ export const BattleAbilities: {[k: string]: ModdedAbilityData} = {
...
+					id: move.id,
+					pp: move.pp,
+					maxpp: move.pp,
+					target: move.target,
+					disabled: false,
+					used: false,
+					virtual: true,
+			};
+			move = this.dex.getMove(typeMap[newTypes[1]]);
+			pokemon.moveSlots[2] = {
```
Comment (ghost) [inline data/mods/ssb/abilities.ts:194]: afaik, you'll be overwriting your custom move

Comment (tmagicturtle) [reply data/mods/ssb/abilities.ts:194]: that's correct, the custom move is in slot 2.
that's what line 182 (pokemon.moveSlots[3] = pokemon.moveSlots[1];) is for, to put the custom move at the end

Comment (tmagicturtle) [reply data/mods/ssb/abilities.ts:194]: would it be better to put placeholder moves in slot 1 & 2 and overwrite them?

Comment (ghost) [reply data/mods/ssb/abilities.ts:194]: ah,
I think you can cut down on hacking by using the unshift array method (see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/unshift).

Comment (ghost) [reply data/mods/ssb/abilities.ts:194]: if it doesnt work i can just merge as is

**Comment by ghost** [comment]
I’m scared about the potential of this code but...

**Pull request merged**